modules/govuk/manifests/node/s_mirrorer.pp

```diff
@@ -7,30 +7,20 @@
 # [*daily_queue_purge*]
 #   This purges the rabbitmq queue of all messages. Use with caution!
 #
-class govuk::node::s_mirrorer (
-  $daily_queue_purge = false,
-) inherits govuk::node::s_base {
+class govuk::node::s_mirrorer inherits govuk::node::s_base {
   include govuk_crawler
   include govuk_rabbitmq
   include nginx
 
-  if $daily_queue_purge {
-    # Create a file that only root can read so not to expose the rabbitmq root password.
-    file { '/root/purge_govuk_crawler_queue.sh':
-      ensure  => present,
-      owner   => 'root',
-      group   => 'root',
-      mode    => '0700',
-      content => "/usr/local/bin/rabbitmqadmin --username=root --password=${::govuk_rabbitmq::root_password} purge queue name=govuk_crawler_queue",
-      require => Class['govuk_rabbitmq'],
-    }
+  # Create a file that only root can read so not to expose the rabbitmq root password.
+  file { '/root/purge_govuk_crawler_queue.sh':
+    ensure  => 'absent',
```

Comment:
I'm not sure if lint should complain with the extra space in the arrow alignment , checks are passing so it should be ok